geemusic/intents/playback.py

```diff
@@ -81,6 +82,25 @@ def prev_song():
 
         return audio().play(stream_url)
 
+@ask.intent("AMAZON.ShuffleOnIntent")
+def shuffle_tracks():
+    shuffled_songs = queue.song_ids
+    
+    if shuffled_songs == [] or None:
+        return statement("There are no songs to shuffle.")
+    else:
```

Comment:
you can remove this else, as you're `return`ing above which will prevent the function from moving on if that `if` branch is taken.